src/routes/index.ts

```diff
@@ -46,6 +47,7 @@ export default async function (fastify: FastifyInstance): Promise<void> {
   fastify.register(redirects);
   fastify.register(automations, { prefix: '/auto' });
   fastify.register(sitemaps, { prefix: '/sitemaps' });
+  fastify.register(slack, { prefix: '/slack' });
```

Comment:
Wondering if we want it under `integrations/slack` or `webhooks/slack`?

Reply:
Its not only webhook, but could be `/integrations/`, will adjust to that 👌

Reply:
I second the `/integrations/` directory